### DISCUSSION TITLE
A way to make subscription groups in pc and import it in Newpipe?

### DISCUSSION THREAD

**Discussion by CrocodileDuty** [TeamNewPipe/NewPipe#11459] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Instead of making the groups and adding channels to them, can we make the file in pc itself?
If yes then what should be the format? Is there any tutorial?

**Comment by asandikci** [comment]
Idk the answer but maybe you could search for related projects:
- BendyStraw
- FreeTube
- UniversalPipeWrench

**Comment by b1k3rdude** [comment]
@sonikun - https://www.thatkarlsmith.com/export-your-youtube-subscriptions/
